types/fusebit__add-on-sdk/index.d.ts

```diff
@@ -0,0 +1,110 @@
+// Type definitions for @fusebit/add-on-sdk 3.1
+// Project: http://fusebit.io/
+// Definitions by: Andrew Miller <https://github.com/andrewrmiller>
+//                 Trent Hashimoto <https://github.com/hashiphoto>
+//                 Andy Dam <https://github.com/andydam>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * Method of invocation for the call coming into Fusebit.
+ */
+export enum Method {
```

Comment:
do you folks need an enum? Could usual union be OK?
```ts
export type Method = 'GET' | 'POST' ...
```
if this is a real const, just create and use const:
https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums

Reply:
Union works.  Updated.